### PR TITLE
Add Synapse job to update the person_id in ecr data based on MPI

### PR DESCRIPTION
**Pull request by m-goggins**
We need to ensure that the values in the person_id column in the eCR data can be refreshed based on changes in the MPI. To do this we need a Synapse job that will join the eCR datastore with the patient table in the MPI on patient_id.

Fixes # [502](https://app.zenhub.com/workspaces/dibbs-63f7aa3e1ecdbb0011edb299/issues/gh/cdcgov/phdi/502)

### CODE REVIEW

File: scripts/Synapse/updateECRDataStorePersonID.ipynb [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        "        return AccessToken(\n",
+        "            token=mssparkutils.credentials.getToken(audience=\"vault\"),\n",
+        "            expires_on=int(time.time())+60*10 # some random time in future... synapse doesn't document how to get the actual time\n",
+        "        )\n",
+        "\n",
+        "credential = ManagedIdentityCredential()\n",
+        "credential._credential = spoof_token() # monkey-patch the contents of the private `_credential`\n",
+        "\n",
+        "# Set your Key Vault information\n",
+        "KEY_VAULT_URL = \"https://devvault9d194c64.vault.azure.net\"\n",
```
Comment (emmastephenson) [inline scripts/Synapse/updateECRDataStorePersonID.ipynb:72]: Should this be an environment variable of some sort? I'm not sure LAC will have exactly the same vault id.

**Review by emmastephenson — COMMENTED** [review]
Largely looks great! Just one question about hardcoding some URLs.

File: scripts/Synapse/updateECRDataStorePersonID.ipynb (hunk 1 above)
Comment (m-goggins) [reply scripts/Synapse/updateECRDataStorePersonID.ipynb:72]: After chatting with Dan this morning, we decided that we need to address this but it's out of scope for this ticket/PR.

**Review by emmastephenson — APPROVED** [review]
LGTM!

**Pull request merged**